integration_test.go

```diff
@@ -285,35 +280,21 @@ func TestIntegrationCartographer(t *testing.T) {
 			mode:        cartofacade.MappingMode,
 			subAlgo:     viamcartographer.Dim2d,
 		},
-		{
-			description: "replay sensor localizing mode 2D",
-			replay:      true,
-			imuEnabled:  false,
-			mode:        cartofacade.LocalizingMode,
-			subAlgo:     viamcartographer.Dim2d,
-		},
 		{
 			description: "replay sensor updating mode 2D",
 			replay:      true,
 			imuEnabled:  false,
 			mode:        cartofacade.UpdatingMode,
 			subAlgo:     viamcartographer.Dim2d,
 		},
-		// Live + imu sensor
+		// Replay with imu sensor
 		{
 			description: "replay with imu sensor mapping mode 2D",
 			replay:      true,
 			imuEnabled:  true,
 			mode:        cartofacade.MappingMode,
 			subAlgo:     viamcartographer.Dim2d,
 		},
-		{
```

Comment:
[note] Had to remove replay localizing tests due to error:

` 'SLAM Service configuration error: camera[data_freq_hz] and enable_mapping = false. localization in offline mode not supported.'`

Reply:
Not too sure whats going on with this error tbh, unless all of our replay camera tests are assumed to be offline mode only in our tests? 

if this is the correct behavior, should we include tests to make sure we hit this error?

Reply:
I think we should only be in offline mode if`camera[data_freq_hz] = 0` but I would expect it to be the default value here

Reply:
Should replay sensors have a non-zero data_freq_hz?

Reply:
for live mode yes, offline mode they should be set to 0

Reply:
Done